### PR TITLE
Add metadatas for stops needing a phone call to book a trip

### LINKED ISSUE #102: Add a metadata if the dataset contains pick_up/drop_off that need calling the agency
It could be useful to add a metadata ([like those](https://github.com/etalab/transport-validator/blob/c9e71b0e5a30f8975543c710bd08b4a8554a9665/src/metadatas.rs#L91-L98)) telling if the gtfs has some pick_up/drop_off that need a phone call ( [pick_up_type/drip_off_type = 2](https://gtfs.org/reference/static#stop_timestxt)), and maybe if it needs to be coordinated with the driver too ? (pick_up_type/drip_off_type = 3)

### PR DESCRIPTION
**Pull request by fchabouis**
I added two fields in the metadatas :
- some_stops_need_phone_agency corresponding to pickup_type or drop_off_type == 2
- some_stops_need_phone_driver corresponding to pickup_type or drop_off_type == 3

closes #102

### CODE REVIEW

File: src/metadatas.rs [hunk 1]
```diff
@@ -15,9 +15,14 @@ pub struct Metadata {
     pub issues_count: std::collections::BTreeMap<IssueType, usize>,
     pub has_fares: bool,
     pub has_shapes: bool,
+    // some stops have a pickup_type or drop_off_type equal to "ArrangeByPhone"
+    pub some_stops_need_phone_agency: bool,
```
Comment (thbar) [inline src/metadatas.rs:19]: I wonder if naming after the original property would help understanding the codebase later?

E.g. `some_stops_need_arrange_by_phone`, `some_stops_need_coordinate_with_driver`

**Review by thbar — APPROVED** [review]
Looks good to me!

I btw wonder (but no need to change yet, I don't know the code enough to have an opinion) if using a lot of fixtures is nice to have long term or not, curious to have the maintainers opinion on this later :smile:

File: src/metadatas.rs [hunk 2]
```diff
@@ -15,9 +15,14 @@ pub struct Metadata {
...
+    // some stops have a pickup_type or drop_off_type equal to "CoordinateWithDriver"
+    pub some_stops_need_phone_driver: bool,
 }
 
 pub fn extract_metadata(gtfs: &gtfs_structures::RawGtfs) -> Metadata {
+    use gtfs_structures::PickupDropOffType::*;
```
Comment (antoine-de) [inline src/metadatas.rs]: I know it's done below, but I don't like glob import, I think it's better to just
`use gtfs_structures::PickupDropOffType` and then `PickupDropOffType::ArrangeByPhone`

**Review by antoine-de — APPROVED** [review]
nice!

File: src/metadatas.rs (hunk 1 above)
Comment (antoine-de) [reply src/metadatas.rs:19]: hum not a strong opinion on this, but I find the names as they are quite understandable.

**Comment by antoine-de** [comment]
for the multiplication of tests it's a tough question, my own heuristic for simple stuff is to try to change an existing fixture and if it breaks anything I add a new one :stuck_out_tongue_winking_eye:

File: src/metadatas.rs (hunk 2 above)
Comment (fchabouis) [reply src/metadatas.rs]: ok j'avais effectivement copié sur l'autre ligne ;)

**Pull request merged**